src/server_manager/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "rootDir": ".",
+  "include": [
+    "**/*.ts"
```

Comment:
You could actually use webpack, and make each test file an entrypoint, outputting them with the file-loader. This way you wouldn't have to worry about the exclude below.

Reply:
Good suggestion. I have a prototype, but we still have to exclude the web app tests from the webpack build. Happy to follow up on this.